screens/Reader.js

```diff
@@ -558,19 +558,19 @@ class Reader extends React.Component {
     });
     return (
       <View
-        style={[
-          styles.container,
-          nightMode && { backgroundColor: "#000" },
-        ]}
+        style={[styles.container, nightMode && { backgroundColor: "#000" }]}
         // eslint-disable-next-line react/jsx-no-bind
```

Comment:
veerji I think you have already disabled this rule globally in a recent PR and it's no longer needed to put in files explicitly. Please correct me if I'm wrong

Reply:
Yes, But the PR is not merged yet. so that's why I have added it in this PR.